packages/mui-system/src/palette.js

```diff
@@ -11,20 +11,20 @@ function transform(value, userValue) {
 export const color = style({
```

Comment:
**Off-topic**

I think that we could move toward moving everything inside this new configuration object. It's what styled system did https://github.com/styled-system/styled-system/blob/6d822f27a486e5e2bc1cae1f44dd151ce874a863/packages/color/src/index.js#L3.

cc @mj12albert as I have seen you passionate about this topic, in case you would like to influence the direction with the system.